lib/standard/bytes.nit

```diff
@@ -46,6 +46,27 @@ class Bytes
 		init(ns, 0, 0)
 	end
 
+	# Create a new `Bytes` instance from a String's bytes
+	#
+	# ~~~nit
+	# assert (new Bytes.from_text("String")) == [83u8, 116u8, 114u8, 105u8, 110u8, 103u8]
```

Comment:
isn't a `String::to_bytes` more easier to use?

~~~nit
assert "String".to_bytes isa Bytes
assert "String".to_bytes == [83u8, 116u8, 114u8, 105u8, 110u8, 103u8]
~~~